release.sh

```diff
@@ -18,18 +19,18 @@ doit() {
   git pull
```

Comment:
You can also make this `git pull --ff-only` just in case, but not really necessary.